virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc

```diff
@@ -10,8 +10,10 @@ You can configure shared disks to allow multiple virtual machines (VMs) to share
 
 You configure disk sharing by exposing the storage as either of these types:
 
-* An ordinary virtual machine disk
-* A logical unit number (LUN) device with an iSCSi connection and raw device mapping, as required for Windows Failover Clustering for shared volumes
+* An ordinary VM disk
+* A logical unit number (LUN) disk with an iSCSi connection and raw device mapping, as required for Windows Failover Clustering for shared volumes
+
+As you configure disk sharing, you can also set an error policy for each ordinary VM disk or LUN disk. The error policy controls how the hypervisor behaves when an input/output error occurs on a disk Read or Write.
```

Comment:
ISG: Do not use _as_ to mean "because" or "while".

```suggestion
While you configure disk sharing, you can also set an error policy for each ordinary VM disk or LUN disk. The error policy controls how the hypervisor behaves when an input/output error occurs on a disk Read or Write.
```
```suggestion
In addition to configuring disk sharing, you can also set an error policy for each ordinary VM disk or LUN disk. The error policy controls how the hypervisor behaves when an input/output error occurs on a disk Read or Write.
```